doc/src/site/sphinx/troubleshooting.rst

```diff
@@ -0,0 +1,101 @@
+Production Checklist & Troubleshooting Guide
+********************************************
+
+Production Checklist
+====================
+
+In this section we provide you the production environment basic settings.
```

Comment:
I prefer: This section shows a list of mandatory requirements and configurations with the aim of to have a clean production environment.